pq-crypto/sike_r2/sidh.c

```diff
@@ -16,16 +17,18 @@ static void init_basis(const digit_t *gen, f2elm_t *XP, f2elm_t *XQ, f2elm_t *XR
 	fpcopy(gen + 5 * NWORDS_FIELD, XR->e[1]);
 }
 
-void random_mod_order_A(unsigned char *random_digits) { // Generation of Alice's secret key
-	                                                    // Outputs random value in [0, 2^eA - 1]
-    get_random_bytes(random_digits, SECRETKEY_A_BYTES);
-	random_digits[SECRETKEY_A_BYTES - 1] &= MASK_ALICE; // Masking last byte
+int random_mod_order_A(unsigned char *random_digits) { // Generation of Alice's secret key
+                                                        // Outputs random value in [0, 2^eA - 1]
+    GUARD_AS_POSIX(get_random_bytes(random_digits, SECRETKEY_A_BYTES));
+    random_digits[SECRETKEY_A_BYTES - 1] &= MASK_ALICE; // Masking last byte
+    return 0;

```

Comment:
`return SUCCESS`

Reply:
Good call (although this will get cleaned up with the pq-crypto migration)